domains/srihari.json

```
@@ -0,0 +1,12 @@
+{
+    "description": "Greetings from Sri Hari!",
+    "repo": "https://github.com/srhr17/srhr17.github.io",
+    "owner": {
+        "username": "srhr17",
+        "email": "srhr1999@gmail.com"
+    },
+    "record": {
+        "URL": "https://srhr17.github.io/srihari-portfolio/"
```

I'd suggest you rename your repo to `srhr17.github.io`. You can take a look at the [github pages guide](https://guides.github.com/features/pages/).

Change this to..
```suggestion
        "CNAME": "srhr17.github.io"
```